README.md

```diff
@@ -2,71 +2,113 @@
 
 This example app uses [Knock](https://knock.app) to power cross channel notifications via email, an in-app feed, and Slack inside of a full-stack Node application, written in [Blitz.js](https://blitzjs.com/). It uses the [Knock Node SDK](https://github.com/knocklabs/knock-node) and [React in-app feed components](https://github.com/knocklabs/react-notification-feed).
 
-You can read more about this example app [in the Knock documentation](https://docs.knock.app/getting-started/example-app).
+You can read more about this example app [in the Knock documentation](https://docs.knock.app/getting-started/example-app#nodejs-example-app).
 
-## Running locally
+## Set up
 
-First, configure the environment variables properly (see the bottom of this README).
+### In the Knock dashboard
 
-Then run:
+[Sign up for a Knock account](https://dashboard.knock.app/signup) to configure channels and workflows for the app to work as expected. Once you've created your Knock account you will need to create channels and workflows to work with your application.
 
-```
-yarn blitz prisma migrate dev
-```
+1. Create an email channel and an in-app feed channel.
+   - You'll need to use an existing email provider like [Postmark](https://postmark.com/) to configure to your email channel in the Knock dashboard.
```

Comment:
Major nit: "to configure to" sounds a bit off, maybe "to configure in" or "to use in"